app/views/advanced_messenger/_html_head.html.erb

```diff
@@ -1,15 +1,243 @@
+<%# ============= Plugin styles  ============= %>
 <% content_for :header_tags do %>
     <%= stylesheet_link_tag 'advanced_messenger', :plugin => 'redmine_advanced_messenger' %>
 <% end %>  
+<%# ============= Global unread notifications indicator located on main title bar  ============= %>
 <%= javascript_tag do%>
-
-    function renderAllUnreadNotificationsIndicator(unreadNotificationsCount) {
-        const color = unreadNotificationsCount > 0 ? "#ff4545" : "#2bff2b";
-        $("#all-unread-notifications-indicator").html("<a href='/my/page'><%=t(:top_app_menu_unread_notifications)%>: <span style='color:" + color + "; font-weight:bold'>" + unreadNotificationsCount + "</span></a>");
+    function renderGlobalUnreadNotificationsIndicator(unreadNotificationsCount) {
+        if ($("#all-unread-notifications-indicator").length > 0) {
+            const color = unreadNotificationsCount > 0 ? "#ff4545" : "#2bff2b";
+            $("#all-unread-notifications-indicator").html("<a href='/my/page'><%=t(:top_app_menu_unread_notifications)%>: <span style='color:" + color + "; font-weight:bold'>" + unreadNotificationsCount + "</span></a>");
+        }
+    }
+    function requestAndRenderGlobalUnreadNotificationsIndicator() {
+        $.ajax({
+            url: '/advanced_messenger/unread_notifications_count', // The URL of your Rails controller action
+            type: 'GET',
+            dataType: 'json',
+            beforeSend: function() { 
+                $("#all-unread-notifications-indicator").addClass('ajax-loading'); 
+            },
+            complete: function() { 
+                $("#all-unread-notifications-indicator").removeClass('ajax-loading'); 
+            },
+            success: function(result) {
+                renderGlobalUnreadNotificationsIndicator(result.count)
+            },
+            error: function(xhr, status, error) {
+                console.error('Error on getting unread_notifications_count');
```

Comment:
Maybe add the `status` and `error` to the console as well.

I'm thinking somehow we should mark the `$("#all-unread-notifications-indicator")` with an error message as well.

What do you think?